mobile/src/main/res/values/strings.xml

```diff
@@ -431,6 +431,8 @@ Good luck, and have fun!
     <string name="pref_consolidate_summary">"Only show the most recent printing of a card"</string>
     <string name="pref_persist_search_title">"Save Search Options"</string>
     <string name="pref_persist_search_summary">"Save search options when leaving the search page"</string>
+    <string name="pref_search_languages_title">"Search Languages"</string>
+	<string name="pref_search_languages_summary">"Languages used for card searches (default: English)"</string>
```

Comment:
If you add a `android:defaultValue` for English you don't need to note it in the name

Reply:
My thinking was, there should be some default behavior, if the user deselects all search languages.
I implemented it to use English.
To tell the user that the default value even in this case is English, I added the note to the preference summary.
This is not clear, if you only have an android:defaultValue, which you can deselect.
To emphasize on this, I thought it's better to have no default value for the preference and removed the android:defaultValue altogether.

But I'm not sure if this is the best way to handle this.
What do you think?

Reply:
Oh, it's a bunch of checkboxes, not a RadioGroup, so the user can deselect them all. The note here makes sense.

I would still recommend having both the note and the `android:defaultValue`. It's more intuitive for a user to see that English is the checked off default when the list is opened.

Reply:
OK, I added it.